perma_web/requirements.txt

```diff
@@ -68,6 +69,7 @@ jsmin==2.0.9
 jsonpatch==0.4            # via internetarchive
 kombu==3.0.37             # via celery
 LinkHeader==0.4.3
+lxml==3.7.3               # via pyquery
```

Comment:
This rings a bell for me as well, but the lxml package appears to compile and run correctly on perma prod, so I think we're good.